### PR TITLE
mupdf: apply CVE-2017-14685 through CVE-2017-14687 patches

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mupdf-tools.rb [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+  revision 1
   head "https://git.ghostscript.com/mupdf.git"
 
+  stable do
+    url "https://mupdf.com/downloads/mupdf-1.11-source.tar.gz"
+    sha256 "209474a80c56a035ce3f4958a63373a96fad75c927c7b1acdc553fc85855f00a"
+
+    # Upstream already. Remove on next stable release.
+    patch do
+      url "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/m/mupdf/mupdf_1.11+ds1-1.1.debian.tar.xz"
```
Comment (ilovezfs) [inline Formula/mupdf-tools.rb]: url & mirror

Comment (DomT4) [reply Formula/mupdf-tools.rb]: Sorry, I was having some tediously slow internet speeds & just wanted to get something pushed 😓. Will sort this shortly.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
Thanks Mr. T

**Comment by DomT4** [comment]
Ta for merging ❤️

**Pull request closed without merging**